cadcutils/cadcutils/net/ws.py

```diff
@@ -686,7 +691,16 @@ def combine_headers(new_headers):
             PUT_TXN_OP: PUT_TXN_COMMIT,
             HTTP_LENGTH: '0'})
         self._get_session().put(url, verify=self.verify, **kwargs)
-        return dest_name, dest_md5
+        self._log_download(src, start, stat_info.st_size)
+        return dest_name, dest_md5, stat_info.st_size
+
+    def _log_download(self, src, start, size):
+        duration = time.time() - start
+        self.logger.info(
+            'Successfully uploaded file {} in {}s '
```

Comment:
Maybe use the word `transferred` instead of `uploaded` if this is a bi-directional message?

Reply:
Doh. Changed method name to `_log_upload` because it's used only in the `upload` method.